devito/arch/archinfo.py

```diff
@@ -664,6 +664,16 @@ def max_mem_trans_size(self, dtype):
         assert self.max_mem_trans_nbytes % np.dtype(dtype).itemsize == 0
         return int(self.max_mem_trans_nbytes / np.dtype(dtype).itemsize)
 
+    def limits(self, compiler=None, language=None):
+        """
+        Return the architecture-specific limits for the given compiler and
+        language.
+        """
+        return {
+            'max-par-dims': np.inf,
```

Comment:
I think `np.inf` might be problematic if used for indexing

Reply:
and indeed one test is failing because of this :D